src/main/java/frc/robot/RobotContainer.java

```diff
@@ -21,10 +29,18 @@
 public class RobotContainer {
   // The robot's subsystems and commands are defined here...
   private final DriveBase driveBase = new DriveBase(0, 1, 2, 3); // TODO: remove dummy port values
+  private final Intake intake1 = new Intake(IntakeConstants.INTAKE_1_MOTOR_ID,
```

Comment:
change `intake1` and `intake2` to `intakeFront` and `intakeBack` (or some tightly coupled variation) wherever they appear